### PR TITLE
Embedding-CSP: Changing serialized-source-list to source-list

### PR DESCRIPTION
**Pull request by Sun77789**
@mikewest

### CODE REVIEW

File: embedded/index.src.html [hunk 1]
```diff
@@ -773,19 +771,19 @@ <h4 id="subsume-source-list" algorithm>
 
   3.  If |B| is empty or |A| is `none`, return "`Does Not Subsume`".
 
-  4.  If |directive B| is "`script-src`" and |B| contains a 
+  4.  If |directive B| is "`script-src`" and |B| contains a token that matches a
```
Comment (Sun77789) [inline embedded/index.src.html]: Not sure if this is necessary to specify or we could just say "|B| contains" some keyword expression?

Comment (mikewest) [inline embedded/index.src.html]: It's probably worth adding a definition to the CSP3 spec along the lines of `A <a>directive</a> <dfn for="directive">contains</dfn> a given <a>source expression</a> if the expression is <a for="list">contained</a> by its <a for="directive">value</a>.` and using that throughout.

File: embedded/index.src.html [hunk 2]
```diff
@@ -773,19 +771,19 @@ <h4 id="subsume-source-list" algorithm>
...
       <a grammar>`keyword-source`</a> expression "`strict-dynamic`" but |A| does
       not contain it, return "`Does Not Subsume`".
 
   5.  If |directive B| is "`script-src`" or "`style-src`":
 
-      1.  If |B| contains a <a grammar>`keyword-source`</a> expression 
-          "`unsafe-eval`" but |A| does not contain it, return 
+      1.  If |B| contains a token that matches a <a grammar>`keyword-source`</a>
```
Comment (mikewest) [inline embedded/index.src.html]: Nit: `<a for="list">contains</a>`. Linkify everything! ;)

Comment (mikewest) [inline embedded/index.src.html]: (Here and elsewhere)

File: embedded/index.src.html (hunk 1 above)
Comment (mikewest) [inline embedded/index.src.html]: (Doesn't need to be in this patch)

**Pull request merged**